hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/EventCondition.java

```diff
@@ -84,9 +99,12 @@ public void setExpression(String expression) {
     }
 
     public boolean match(Event value) {
-        if (null == expression || expression.isEmpty() || null == value) {
+        if (null == value) {
             return false;
         }
+        if (null == expression || expression.isEmpty()) {
+            return true;
+        }
         String[] expressions = expression.split(",");
```

Comment:
+1 to better documentation
String manipulations operations are separated by simple quotes.
A complex expression for a type was something interesting as in the same event I would like to have some predicates for the expressions without moving that to several conditions for the same data.
Expressions brought to my mind as the simple solution here, I think.